### ISSUE TITLE
error using php artisan entrust:migration

### ISSUE THREAD

**Issue by luisfer7192** [Zizaco/entrust#620]
When i was installing i had an error using php artisan entrust:migration

`// Create table for associating roles to users (Many-to-Many)
        Schema::create('role_user', function (Blueprint $table) {
            $table->integer('user_id')->unsigned();
            $table->integer('role_id')->unsigned();

            $table->foreign('user_id')->references('id')->on('')
                ->onUpdate('cascade')->onDelete('cascade');
            $table->foreign('role_id')->references('id')->on('roles')
                ->onUpdate('cascade')->onDelete('cascade');

            $table->primary(['user_id', 'role_id']);
        });`

It has an error in `$table->foreign('user_id')->references('id')->on('')
`
it should be `$table->foreign('user_id')->references('id')->on('users')`

**Comment by santigarcor** [comment]
You should try [this](https://github.com/santigarcor/laratrust)

**Comment by luisfer7192** [comment]
Thx, but i don't have an error, i just put the problem.

**Comment by Remade** [comment]
Check that the table name for users' table has been set for Laravel's auth system

**Comment by d-lamers** [comment]
Same issue here

**Comment by musman92** [comment]
see on('') is empty, 
be default its empty due entrust do not know what is the table name of user's table.
so you need to add your table name and problem is solved.

 $table->foreign('user_id')->references('id')->on('')

like.
user's db table name is "users"
it look like
$table->foreign('user_id')->references('id')->on('users')

**Comment by d-lamers** [comment]
Yes I get this, but it makes it not work out of the box. My pull request will fox that issue while still being able to have a different table name. The reason for my concern is that a cms package (for example) could require this package and should be able to intialize it by itself withouth user config.
